### DISCUSSION TITLE
离谱的东西

### DISCUSSION THREAD

**Discussion by zsr-lukezhang** [win12-online/win12#496] [General]
[Some ~~interesting~~  **horrible** things](https://felixng1988.github.io/shop-gnu_linux_wipeout_spray)

**Comment by zsr-lukezhang** [comment]
@lingbopro @User782Tec @NB-Group @tjy-gitnub 我无语。。。

**Reply by lingbopro** [reply]
wat?

**Reply by lingbopro** [reply]
什么啊（我没看

**Reply by zsr-lukezhang** [reply]
> 什么啊（我没看

自己看

**Reply by zsr-lukezhang** [reply]
> > 什么啊（我没看
> 
> 自己看

做好心理准备

**Reply by lingbopro** [reply]
大概是什么

（听你这么一说更不敢看了

**Reply by zsr-lukezhang** [reply]
反对某个公司

**Reply by ProgramCX** [reply]
> 大概是什么
> 
> （听你这么一说更不敢看了

还好吧，就是搞笑

**Comment by lingbopro** [comment]
草，神金

话说这个东西为什么要建一个讨论

**Reply by zsr-lukezhang** [reply]
不想污染别的讨论

**Reply by ProgramCX** [reply]
不过windows的确臃肿，我看好Linux

**Reply by zsr-lukezhang** [reply]
看后面的
[这个](https://github.com/tjy-gitnub/win12/discussions/496#discussioncomment-11276139)

**Comment by zsr-lukezhang** [comment]
[BILIBILI](https://space.bilibili.com/678665731?spm_id_from=333.337.search-card.all.click)

**Comment by tjy-gitnub** [comment]
赞成。喜欢 linux

**Reply by lingbopro** [reply]
那确实

**Reply by lingbopro** [reply]
就比如782（bushi

**Reply by Mbilse** [reply]
> 就比如782

就比如我

**Comment by Mbilse** [comment]
> [Some ~~interesting~~  **horrible** things](https://felixng1988.github.io/shop-gnu_linux_wipeout_spray)

404了
